src/arrays/programs_involving_multidimensional_data/easy/q8/EvenOrOdd.java

```diff
@@ -0,0 +1,33 @@
+package arrays.programs_involving_multidimensional_data.easy.q8;
+
+/*
+Write a Java program that generates a 2D array of random integers of size 5 by 5, with the range of values from 1-1000
```

Comment:
Change to:
"Make a Java program that generates a 5 by 5 2D array of random integers ranging from 1-1000 (inclusive on both ends), and print it. Then, multiply all the elements of the array by 5 if the element is even and by 3 if the element is odd. Lastly, print out the array at the end of the program with each row of numbers on a separate line."